app/src/main/java/io/homeassistant/companion/android/notifications/MessagingManager.kt

```diff
@@ -377,6 +379,19 @@ class MessagingManager @Inject constructor(
                             }
                         }
                     }
+                    COMMAND_BEACON_MONITOR -> {
+                        if (!jsonData[TITLE].isNullOrEmpty() && jsonData[TITLE] in ENABLE_COMMANDS)
```

Comment:
We also need to switch to `jsonData[COMMAND]` here as well, just like the other commands above/below :)

Reply:
ups I overlooked that